app/views/site/contribute.erb

```diff
@@ -0,0 +1,168 @@
+
+          <p>
+            Another great way to add new exercises is port an existing exercise to one of the other languages. This is easier than making up an exercise from scratch, since other people have thought through which test cases are needed.
+          </p>
+
+          <p>
+            Track maintainers are more than happy to give you all the support you need to successfully contribute.
+          </p>
+
+          <p>
+            Check out the <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track">detailed guide</a> if you want to give it a try!
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Review Pull Requests</h2>
+
+          <p>
+            We want the quality of the exercises to be as high as possible (and reviewing pull requests is a great way to both learn and teach a programming language).
+          </p>
+
+          <p>
+            We look at all sorts of things: style, idiomatic use of language features, naming, missing test cases, and redundant test cases. We also try to avoid forcing the solution to take a particular approach. Subscribe to the language track repositories of your favorite languages so you get notifications about new issues and pull requests. The track maintainers will greatly appreciate your help!
+          </p>
+
+          <p>
+             You can search for language track repositories <a href="https://github.com/exercism/">here</a>.
+          </p>
+        </div>
+
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Canonicalize Data</h2>
+
+          <p>
+            Different implementations of the same exercise often drift apart as people solving them discover different, interesting edge cases. Defining a canonical set of test data helps improve the quality all the implementations, and makes it easier to port the exercise to new languages.
+          </p>
+
+          <p>
+            Do a survey of the existing implementations, and decide on a good set of test data. Submit a pull request to the <a href="https://github.com/exercism/x-common">metadata repository</a> suggesting a good set of test data, and the track maintainers will be delighted to discuss it with you.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Help Triage Issues</h2>
+
+          <p>
+            We've got a huge number of old issues that aren't particularly actionable, and new issues come in all the time.
+          </p>
+
+          <p>
+            Is an issue no longer relevant? Can it be closed? Do we need more information? If it's a bug, is it reproducible? If it's a discussion and we need to make some sort of decision, then it should be labeled as such.
```

Comment:
So far this is the only section without links, what do you think of directing people to major repository issues? I think they are `x-common` and `exercism.io`?